Modules/System/Assisted Setup/src/AssistedSetup.Codeunit.al

```diff
@@ -89,6 +89,15 @@ codeunit 3725 "Assisted Setup"
         AssistedSetupImpl.Complete(ExtensionID, PageID);
     end;
 
+    /// <summary>Sets the status of the assisted setup to uncomplete.</summary>
+    /// <param name="ExtensionID">The app ID of the extension to which the setup belongs.</param>
+    /// <param name="PageID">The ID of the page to open when the user clicks the setup.</param>
+    /// <returns>True if the assisted setup is uncomplete now.</returns>
```

Comment:
The function does not seem like it it returning anything. Please remove comment.